src/page/ConfigPageLayout/tabs/TerraformTab.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useEffect } from 'react';
+import { GrafanaTheme2 } from '@grafana/data';
+import { Alert, Text, TextLink, useStyles2 } from '@grafana/ui';
+import { css } from '@emotion/css';
+import { generateRoutePath } from 'routes';
+
+import { ROUTES } from 'types';
+import { FaroEvent, reportEvent } from 'faro';
+import { useTerraformConfig } from 'hooks/useTerraformConfig';
+import { Clipboard } from 'components/Clipboard';
+
+import { ConfigContent } from '../ConfigContent';
+
+export function TerraformTab() {
+  const { config, checkCommands, probeCommands, error, isLoading } = useTerraformConfig();
+  const styles = useStyles2(getStyles);
+  useEffect(() => {
+    reportEvent(FaroEvent.SHOW_TERRAFORM_CONFIG);
+  }, []);
+
+  if (isLoading) {
+    return <ConfigContent loading={isLoading} title="Terraform config" />;
+  }
+
+  return (
+    <ConfigContent title="Terraform config">
+      {error && <Alert title={error.message} />}
+      <p>
+        You can manage Synthetic monitoring checks using Terraform as well as export your current checks as
+        configuration.
+      </p>
+
+      <ConfigContent.Section title="Prerequisites">
+        <div>
+          <TextLink href="https://grafana.com/docs/grafana/latest/administration/service-accounts/" external>
+            Grafana API key
+          </TextLink>
+        </div>
+
+        <div>
+          <TextLink href={`${generateRoutePath(ROUTES.Config)}/access-tokens`}>
+            Synthetic monitoring access token
+          </TextLink>
+        </div>
+      </ConfigContent.Section>
+
+      <ConfigContent.Section title="Exported config">
+        <Alert title="Terraform and JSON" severity="info">
+          The exported config is using{' '}
+          <a href="https://www.terraform.io/docs/language/syntax/json.html">Terraform JSON syntax</a>. You can place
```

Comment:
It's hard to notice this is a link:
![image](https://github.com/user-attachments/assets/af6e4e2a-f90f-4785-9d87-314f8e79dcaa)

Why not use `TextLink` instead?
```
<TextLink href="https://www.terraform.io/docs/language/syntax/json.html" external={true}>
          Terraform JSON syntax
</TextLink>
```

Reply:
> Why not use TextLink instead?
We'd have to ask the author of the code :)
It's the same old `Alert` as before. Good catch though, I will update it 👌🏻